packages/react/src/components/Tile/Tile.js

```diff
@@ -174,9 +195,11 @@ export class ClickableTile extends Component {
       <Link
         href={href}
         className={classes}
-        {...other}
-        onClick={this.handleClick}
-        onKeyDown={this.handleKeyDown}>
+        {...rest}
+        // TODO: replace with onClick when handleClick prop is deprecated
+        onClick={composeEventHandlers([this.onClick, this.handleClick])}
+        // TODO: replace with onKeyDown when handleKeyDown prop is deprecated
+        onKeyDown={composeEventHandlers([this.onKeyDown, this.handleKeyDown])}>
```

Comment:
I think these handlers should stay the same since `this.handleClick` is now conditionally calling `props.handleClick` or `props.onClick`. I couldn't find `this.onClick` defined on the class so let me know if I'm missing something 🤔